src/rt/model.c

```diff
@@ -3274,3 +3300,10 @@ void x_unreachable(tree_t where)
    else
       jit_msg(NULL, DIAG_FATAL, "executed unreachable instruction");
 }
+
+void x_cover_setup_toggle_cb(sig_shared_t *ss, int32_t *toggle_mask)
```

Comment:
This can go in `cover.c`, no need to add it here.

Reply:
OK, moved

Reply:
OK, moved